app/jobs/remove_from_metadata_cloud_csv_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class RemoveFromMetadataCloudCsvJob < ApplicationJob
```

Comment:
Can we drop the MetadataCloud reference here?  It's a misnomer in the other job as well.  Something like "DeleteObjectsJob" would be fine.

Reply:
Yep, no problem! Thanks for the feedback